lib/wikidata_lookup.rb

```diff
@@ -148,12 +148,12 @@ def other_fields_for(result)
   end
 
   private
-  WDQ_URL = 'https://wdq.wmflabs.org/api'
+  WDQ_URL = 'https://query.wikidata.org/sparql'
 
   def wdq(query)
```

Comment:
probably better to rename the method and URL etc, as WDQ is the name of the specific Wikidata tool, rather than just a generic concept, so this could be quite confusing later.